src/common/config.cc

```diff
@@ -33,6 +33,7 @@
 #include <sstream>
 #include <stdlib.h>
 #include <string.h>
+#include <libgen.h>
```

Comment:
@ukernel Including libgen.h this way will result in complaints on some other platforms. Hence I included it in an FreebsD conditional.